.github/workflows/lint.yml

```diff
@@ -27,3 +27,25 @@ jobs:
         shell: bash
       - run: go version
       - run: make lint
+
+  prettify:
+    name: Prettify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        if: github.event_name == 'pull_request'
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+      - uses: actions/checkout@v3
+        if: github.event_name == 'push'
+        with:
+          fetch-depth: 0
+      - name: Install npm v8
+        run: npm i -g npm@8
+      - name: Prettify markdown code
```

Comment:
Will this fail if there are code changes? That's the intent of CI (ensure code is prettified).

Reply:
Nope, this linter will not show the difference, but immediately starts changing specified files according to the standard. Of course, you can also add a job simply with a `--check` flag, so that it checks for an error if it indicates the difference. But when I read the description of the issue it was about auto-formatting the code in the pipeline, if I'm wrong, please correct me

Reply:
I think the developer should be making changes and ensuring code is linted (you should make it easy).
CI is here to enforce it, so this should only run a check.

Reply:
Origin issue https://github.com/opensearch-project/opensearch-go/issues/260 says that there must be "A linter that auto-formats code."
Your comments confused me((
Do you need a linter that just checks and shows the user where something is wrong, or will it check and auto-format the code?

Reply:
1. Developers need an explanation of how to run the linter when making code changes. This can be done command-line/integrated into the IDE, etc, instructions should be added to DEVELOPER_GUIIDE. 
2. CI needs to verify that the code is formatted according to a standard. Thus when one makes pull requests CI runs the linter checks and fails if conventions aren't followed.

I don't know what the Go standard linter is, but let's use it. For Ruby it's RuboCop, https://github.com/rubocop/rubocop, for example.